### PR TITLE
TIP-901: update dependencies

### PR DESCRIPTION
**Pull request by jjanvier**
This PR simply bumps a few dependencies to the last patches.
This is necessary in order to be able to upgrade Symfony to the version 4.

See https://github.com/akeneo/pim-community-dev/pull/10395

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -33,8 +33,6 @@ public function registerBundles()
         $bundles = $this->registerProjectBundles();
 
         if (in_array($this->getEnvironment(), array('dev', 'test', 'behat'))) {
-            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
```
Comment (arnolanglade) [inline app/AppKernel.php]: Are you sure? is it needed to generate bootstrap.php.cache ?

File: app/AppKernel.php [hunk 2]
```diff
@@ -33,8 +33,6 @@ public function registerBundles()
...
-            $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
```
Comment (arnolanglade) [inline app/AppKernel.php]: :+1:

File: composer.json [hunk 3]
```diff
@@ -50,38 +50,36 @@
...
-        "sensio/framework-extra-bundle": "3.0.26",
-        "sensio/generator-bundle": "3.1.6",
+        "sensio/framework-extra-bundle": "5.2.4",
         "symfony/monolog-bundle": "3.1.2",
-        "symfony/swiftmailer-bundle": "3.0.3",
-        "symfony/security-acl": "3.0.0",
+        "symfony/swiftmailer-bundle": "3.2.4",
+        "symfony/security-acl": "3.0.2",
         "symfony/symfony": "3.4.28",
         "symfony/thanks": "^1.0",
```
Comment (arnolanglade) [inline composer.json]: We should remove this dependencies

File: app/AppKernel.php (hunk 1 above)
Comment (jjanvier) [reply app/AppKernel.php]: @arnolanglade this bundle is not compatible with Symfony 4

Comment (jjanvier) [reply app/AppKernel.php]: cf https://github.com/sensiolabs/SensioDistributionBundle/blob/master/composer.json

Comment (arnolanglade) [reply app/AppKernel.php]: yes but do we need to until dumping Symfony to V4?

File: composer.json (hunk 3 above)
Comment (jjanvier) [reply composer.json]: I'll do in another PR.
The CI is green for now

File: app/AppKernel.php (hunk 1 above)
Comment (jjanvier) [reply app/AppKernel.php]: what is used `bootstrap.php.cache` for? 
if the CI is green, then it's OK no?

File: composer.json [hunk 4]
```diff
@@ -50,38 +50,36 @@
         "doctrine/cache": "1.8.0",
         "doctrine/collections": "1.5.0",
         "doctrine/common": "2.10.0",
-        "doctrine/data-fixtures": "1.2.2",
```
Comment (fitn) [inline composer.json]: Do we still use it? Same for doctrine-fixtures-bundle?
That's not directly linked to your PR anyway ;)

Comment (jjanvier) [reply composer.json]: I'll check !
and change in another PR if needed

**Pull request merged**